1주차/스택.py

```diff
@@ -0,0 +1,40 @@
+def push(X):
+    my_stack.append(X)
+
+def top():
+    if len(my_stack) >= 1:
```

Comment:
```suggestion
    if my_stack:
```
빈 리스트는 False, 비어있지 않은 리스트는 True로 취급되니 이렇게 쓸 수도 있다는거 알아두세요